nnbuilder-api/src/main/proto/tasksqueue.proto

```diff
@@ -0,0 +1,43 @@
+syntax = "proto3";
+
+package org.hse.nnbuilder.services;
+
+enum TaskName {
+  LearnNN = 0;
+  ApplyToData = 1;
+}
+
+enum TaskStatus {
+  HaveNotStarted = 0;
+  Processing = 1;
+  Done = 2;
+}
+
+service TasksQueueService {
+  rpc createtask (TaskCreationRequest) returns (TaskCreationResponse);
```

Comment:
I still think it's better to name the request/response as call + Request/Response, e.g. here
`rpc createTask (TaskCreateRequest) returns (TaskCreateResponse)`
even though your way may be more correct grammatically, this way makes it easier to uniformly identify the request and response objects